db/migrate/20230812061445_remove_reconfirmable_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveReconfirmableFromUsers < ActiveRecord::Migration[7.0]
```

Comment:
If this column isn't needed, you can delete the migration that adds it and the one that removes it.